pkg/apis/sriovnetwork/v1/sriovnetworknodepolicy_types.go

```diff
@@ -41,7 +41,6 @@ type SriovNetworkNicSelector struct {
 	// +kubebuilder:validation:Enum={"8086","15b3"}
 	// The vendor hex code of SR-IoV device. Allowed value "8086", "15b3".
 	Vendor      string   `json:"vendor,omitempty"`
-	// +kubebuilder:validation:Enum=1583,158b,10fb,1015,1017
```

Comment:
We can keep this line by changing the formate to `Enum={"1583","158b","10fb","1015","1017"}`

Reply:
fixed